### PR TITLE
Adding VGG to model zoo

### PR DESCRIPTION
**Pull request by abhinavs95**
Adding VGG models to model zoo - 16, 16_bn, 19, 19_bn. Models are trained with Gluon using script `train_imagenet.py` from gluoncv.

Raised PR in dmlc/web-data for adding the train commands - [PR](https://github.com/dmlc/web-data/pull/100)

### CODE REVIEW

**Comment by abhinavs95** [comment]
@hetong007 could you please have a look? Thanks!

**Comment by mli** [comment]
Job PR-217-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-217/1/index.html 
Code coverage of this PR:  vs. Master:

File: docs/model_zoo/index.rst [hunk 1]
```diff
@@ -102,6 +102,14 @@ Besides the listed, we provide more models trained on ImageNet in the upstream
...
 | MobileNetV2_0.25 [7]_ | 50.74  | 74.56  | `shell script <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/classification/imagenet/mobilenetv2_0.25.sh>`_  | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/classification/imagenet/mobilenetv2_0.25.log>`_     |
 +-----------------------+--------+--------+------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| VGG16 [9]_            | 72.38  | 91.02  | `shell script <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/classification/imagenet/vgg16.sh>`_             | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/classification/imagenet/vgg16.log>`_                |
```
Comment (hetong007) [inline docs/model_zoo/index.rst]: please leave log empty for now, as we don't have them yet.

Comment (abhinavs95) [reply docs/model_zoo/index.rst]: done

**Comment by mli** [comment]
Job PR-217-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-217/2/index.html 
Code coverage of this PR:  vs. Master:

**Comment by hetong007** [comment]
Pending for https://github.com/dmlc/web-data/pull/100 .

**Pull request merged**